src/utils/monitoring-sites-data.js

```diff
@@ -74,6 +74,25 @@ const fishPopulationFreshwaterSites = ([sw, ne]) => {
   );
 };
 
+const bristolBaseUrl = `https://opendata.bristol.gov.uk/api/records/1.0/search/?dataset=surface-water-quality&q=&sort=datetime`;
+
+const bristolWaterQualityQuery = (map) => {
+  const { sw, nw, ne, se } = getBboxCorners(map);
+
+  return (
+    `${bristolBaseUrl}&facet=siteid&rows=30` +
```

Comment:
I think it's marginally faster to get more rows and filter through them than to get fewer, which might mean we have to make another request - ⬇️  line 147